trade-finance-manager-api/src/v1/controllers/acbs.controller.js

```diff
@@ -62,41 +65,43 @@ const updateIssuedFacilityAcbs = ({ facilityId, issuedFacilityMaster }) =>
   tfmController.updateFacilityAcbs(facilityId, { issuedFacilityMaster });
 
 const checkAzureAcbsFunction = async () => {
+  try {
   // Fetch outstanding functions
-
-  const collection = await db.getCollection('durable-functions-log');
-  const runningTasks = await collection.find({
-    type: 'ACBS',
-    status: 'Running',
-  }).toArray();
-
-  const tasks = runningTasks.map(({ acbsTaskLinks = {} }) => api.getFunctionsAPI(acbsTaskLinks.statusQueryGetUri));
-  const taskList = await Promise.all(tasks);
-
-  taskList.forEach(async (task) => {
-    // eslint-disable-next-line no-underscore-dangle
-    // Update
-    if (task.runtimeStatus !== 'Running') {
-      await collection.findOneAndUpdate(
-        { instanceId: task.instanceId },
-        $.flatten({
-          status: task.runtimeStatus,
-          acbsTaskResult: task,
-        }),
-      );
-    }
-
-    if (task.runtimeStatus === 'Completed') {
-      switch (task.name) {
-        case 'acbs-issue-facility':
-          await updateIssuedFacilityAcbs(task.output);
-          break;
-
-        default:
-          await updateDealAcbs(task.output);
+    const collection = await db.getCollection('durable-functions-log');
+    const runningTasks = await collection.find({
+      type: 'ACBS',
+      status: 'Running',
+    }).toArray();
+    const tasks = await runningTasks.map(async ({ acbsTaskLinks = {} }) =>
+      api.getFunctionsAPI(CONSTANTS.DURABLE_FUNCTIONS.TYPE.ACBS, acbsTaskLinks.statusQueryGetUri));
```

Comment:
Nice - was the issue here that `acbsTaskLinks.statusQueryGetUri` wasn't returning a uri or invalid one?

Reply:
The issue was first argument was missing and regex needs ternary operator for safety.